src/CreeDictionary/API/search/presentation.py

```diff
@@ -141,7 +158,28 @@ def safe_partition_analysis(analysis: ConcatAnalysis):
 
 def replace_user_friendly_tags(fst_tags: List[FSTTag]) -> List[Label]:
     """replace fst-tags to cute ones"""
-    return LABELS.english.get_full_relabelling(fst_tags)
+    return read_labels().english.get_full_relabelling(fst_tags)
+
+
+def get_emoji_for_cree_wordclass(word_class: Optional[str]) -> Optional[str]:
+    """
+    Attempts to get an emoji description of the full wordclass.
+    e.g., "👤👵🏽" for "nôhkom"
+    """
+    if word_class is None:
+        return None
+
+    def to_fst_output_style(value):
+        if value[0] == "N":
+            return "+" + "+".join(list(value.upper()))
+        elif value[0] == "V":
+            return "+" + "+".join(["V", value[1:].upper()])
+        else:
+            return "+" + value.title()
+
+    fst_tag_str = to_fst_output_style(word_class).strip("+")
```

Comment:
Why prepend `"+"` to all of these and then call `.strip("+")` on the return?

Reply:
Thanks for pointing this out; I just brought back the code from the git history without looking carefully at what it was doing.